r/configure

```diff
@@ -48,41 +48,41 @@ else
   # Use pkg-config if available
   pkg-config --version >/dev/null 2>&1
   if [ $? -eq 0 ]; then
-    echo "$ pkg-config --cflags --silence-errors ${PKG_CONFIG_NAME}"
+    # echo "$ pkg-config --cflags --silence-errors ${PKG_CONFIG_NAME}"
     PKGCONFIG_CFLAGS=`pkg-config --cflags --silence-errors ${PKG_CONFIG_NAME}`
-    echo "PKGCONFIG_CFLAGS = \"${PKGCONFIG_CFLAGS}\""
+    # echo "PKGCONFIG_CFLAGS = \"${PKGCONFIG_CFLAGS}\""
 
-    echo "$ pkg-config --libs ${PKG_CONFIG_NAME}"
+    # echo "$ pkg-config --libs ${PKG_CONFIG_NAME}"
     PKGCONFIG_LIBS=`pkg-config --libs --silence-errors ${PKG_CONFIG_NAME}`
-    echo "PKGCONFIG_LIBS = \"${PKGCONFIG_LIBS}\""
-    echo ""
+    # echo "PKGCONFIG_LIBS = \"${PKGCONFIG_LIBS}\""
+    # echo ""
   fi
 
   if [ "$PKGCONFIG_CFLAGS" ] || [ "$PKGCONFIG_LIBS" ]; then
-    echo "Found pkg-config cflags and libs!"
+    echo "Arrow C++ libraries found via pkg-config"
     PKG_CFLAGS="$PKGCONFIG_CFLAGS"
     PKG_LIBS=${PKGCONFIG_LIBS}
   else
     if [[ "$OSTYPE" == "darwin"* ]]; then
       if [ "$(command -v brew)" ]; then
-        echo "brew is available"
-        BREWDIR=$(brew --prefix)
-
-        if brew ls --versions apache-arrow > /dev/null; then
-          # Install apache-arrow via Homebrew if we don't already have it
+        if [ "$(brew ls --versions apache-arrow)" == "" ]; then
```

Comment:
Can we use `${PKG_BREW_NAME}` instead of `apache-arrow` here and other `apache-arrow`s too?

Reply:
Done, and also removed the commented echos